.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Run Python Snippets
+
+on:
+  pull_request:
+
+  workflow_dispatch:
+
+jobs:
+  run-tests-doc-snippets:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    name: Test All Python Snippets in Documentation
+    steps:
+      - uses: actions/checkout@v2  # Pulls the gitbook repo and switch to the correct branch
+
+      - uses: ./.github/actions/setup-server
+        timeout-minutes: 5
+
+      - name: Run the Documentation Code Snippets
+        timeout-minutes: 10
+        working-directory: ../aqueduct/regression_tests/tests/gitbook
+        run: python3 test_docs.py --docs-folder ../../../../gitbook
```

Comment:
nit: I would prefer `python3 aqueduct/regression_tests/tests/gitbook/test_docs.py --docs-folder gitbook` for readability (from workflow-directory ..)